share/get.go

```diff
@@ -93,7 +107,7 @@ func GetProof(
 	leaf, total int,
 ) ([]cid.Cid, error) {
 	// request the node
-	nd, err := plugin.GetNode(ctx, bGetter, root)
+	nd, err := ipld.GetNode(ctx, bGetter, root)
```

Comment:
This GetProofs also deserves to be moved in `ipld`

Reply:
good point, move it to same file where GetLeaf is.